### DISCUSSION TITLE
PLAYING BETTER MC with Essential

### DISCUSSION THREAD

**Discussion by MajinPOp** [FabricMC/fabric-api#4291] [Player support]
20:57:23] [Render thread/ERROR]:There's an incompatible mod installed, please remove all extra added mods 
Minecraft Logs

_No response_

### Description

Trying to play Better MC modpack with the Essential mod...I Already DOnwloaded Better MC with Fabric modloader and the Fabric APi as Well but it kept Crashing

**Comment by Poopooracoocoo** [comment]
You have to remove a mod that does a similar thing called "Luna". Although you may like to try e4mc instead of Essential

**Reply by MajinPOp** [reply]
Its still not working

**Reply by Poopooracoocoo** [reply]
It comes from LunaPixel's FancyMenu configs. You can either edit those config files or remove the FancyMenu mod.

Here's how you edit the FancyMenu configs. These steps depend on your Minecraft version. I'll assume 1.21.1 (Better MC [FABRIC] BMC3).

Go to `{modpack}\config\fancymenu\customization` and edit the `error.txt` and remove the following line:
`[loading_requirement:fancymenu_loading_requirement_is_mod_loaded][requirement_mode:if][group:error][req_id:e619ba19-0eb0-4508-a882-634882e022ea1689477516502] = essential`

Edit `{modpack}\config\fancymenu\customization\safety.txt` and remove the following line:
`[loading_requirement:fancymenu_loading_requirement_is_mod_loaded][requirement_mode:if][req_id:5b7e078a-c197-4ae9-befe-841b5ccbdd36-1718111337454] = essential`

Edit `{modpack}\config\fancymenu\customization\title_screen_layout.txt` and remove the following line:
`[loading_requirement:fancymenu_loading_requirement_is_mod_loaded][requirement_mode:if_not][req_id:81094250-6ccd-4c3d-b52b-710d2383cee4-1726464644850] = essential`

If you decide to remove FancyMenu, know that the changelog button comes from Modpack Update Checker.
